tests/FakeItEasy.Specs/FakeItEasy.Specs.ruleset

```diff
@@ -5,4 +5,4 @@
     <Rule Id="CA1003" Action="None" />
     <Rule Id="CA1009" Action="None" />
   </Rules>
-</RuleSet>
\ No newline at end of file
+</RuleSet>

```

Comment:
It is. It just looked odd as a change when nothing else happened here. Not a complaint. Just an observation. It's all goo.d